scripts/perf-cpu.sh

```diff
@@ -15,4 +15,4 @@ export UCX_WARN_UNUSED_ENV_VARS=n
 
 module load julia/1.2.0 openmpi/4.0.1
 
-mpiexec julia --color=no --project $1
+mpiexec julia --color=no --project $*
```

Comment:
```suggestion
mpiexec julia --color=no --project "$@"
```
should correctly handle spaces, see https://stackoverflow.com/a/3898681/392585

Reply:
Ah `bash`...